### DISCUSSION TITLE
Style manager changes are overridden by custom css

### DISCUSSION THREAD

**Discussion by codingmachine16** [GrapesJS/grapesjs#3740] [Q&A]
Hi @artf 

When I add main.css  
    canvas: {
      styles: [
        "https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css",
        "https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.11.2/css/all.css",
        "https://pro.fontawesome.com/releases/v5.10.0/css/all.css",
        "./css/canvas.css",
        "./css/main.css",
      ],
      scripts: [
        "https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js",
        "https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.min.js",
      ],
    },

In main.css there is a rule 

                        #full-header-2 #headerTop p {
                          color: #000000;
                          font-family: 'Livvic', sans-serif;
                          font-size: 11px;
                          text-transform: capitalize;
                          font-weight: bold;
                          padding-top: 5px;
                        }

Now when I try to change color of P from style manager , it is not overriding custom css. Please help in this regard

**Comment by artf** [comment]
Try to use componentFirst option
```js
grapesjs.init({
	...
	selectorManager: { componentFirst: true },
})
```

**Reply by codingmachine16** [reply]
@artf  It does not work

**Reply by Ju99ernaut** [reply]
Looks like this has more to do with selector specificity. For example grapesjs will generate css with the `#full-header-2` selector which is less specific compared to `#full-header-2 #headerTop p`.

**Reply by saudAtIrisdame** [reply]
Hi @codingmachine16 @artf 

I facing the same issue, I have loaded bootstrap. bootstrap styles work in canvas but I can't see them in the style manager.

![](https://user-images.githubusercontent.com/81280927/150540692-624ab958-982e-4ab9-a0aa-4e4b52e898d8.png)

How did you solve this issue? it would be helpful if you share a solution.
